packages/scaffold/src/partials/w3m-account-auth-button/index.ts

```diff
@@ -49,6 +50,18 @@ export class W3mAccountAuthButton extends LitElement {
       RouterController.push('UpdateEmailWallet', { email })
     }
   }
+
+  private getAuthName(email: string) {
+    if (this.socialUsername) {
+      if (this.socialProvider === 'discord' && this.socialUsername.endsWith('0')) {
+        return this.socialUsername.slice(0, -1)
+      }
+
+      return this.socialUsername
+    }
+
+    return email.length > 30 ? `${email.slice(0, -3)}...` : email
```

Comment:
Wouldnt this leave the full email without the .com?

Reply:
If the email is really long it will.. we can also split it in the middle? Don't really know which one I prefer